python/storage/logging.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2022 The KServe Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging.config
+from typing import Optional, Union, Dict
+
+import yaml
+
+KSERVE_LOGLEVEL = "DEBUG"
+KSERVE_LOGGER_NAME = "kserve"
+KSERVE_TRACE_LOGGER_NAME = "kserve.trace"
+KSERVE_LOGGER_FORMAT = (
+    "%(asctime)s.%(msecs)03d %(process)s %(name)s "
+    "%(levelname)s [%(filename)s:%(funcName)s():%(lineno)s] %(message)s"
+)
+KSERVE_TRACE_LOGGER_FORMAT = "%(asctime)s.%(msecs)03d %(name)s %(message)s"
+KSERVE_LOGGER_DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+KSERVE_LOG_CONFIG = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "kserve": {
+            "()": "logging.Formatter",
+            "fmt": KSERVE_LOGGER_FORMAT,
+            "datefmt": KSERVE_LOGGER_DATE_FORMAT,
+        },
+        "kserve_trace": {
```

Comment:
Do we need kserve_trace ?

Reply:
for now, I just replicated the logging.py file.
We can probably clean it.

Reply:
I've cleaned it a few, please take a look